new-explorer-frontend/src/utils/api.js

```diff
@@ -0,0 +1,91 @@
+class Api {
+  constructor({ baseUrl, authorizationCode }) {
+    this._authorization = authorizationCode;
+  }
+
+  _checkResponse(res) {
+    if (!res.ok) {
+      return Promise.reject(res.status);
+    }
+    return res.json();
+  }
+
+  updateUserData(path, name, about) {
+    return fetch(`https://api.aroundori.students.nomoredomainssbs.ru${path}`, {
```

Comment:
`https://api.aroundori.students.nomoredomainssbs.ru` is repeated 7 times. You need to use `baseUrl` from the constructor